crates/kornia-imgproc/src/features.rs

```diff
@@ -68,6 +82,51 @@ pub fn hessian_response(src: &Image<f32, 1>, dst: &mut Image<f32, 1>) -> Result<
     Ok(())
 }
 
+/// Compute the DoG response of an image.
+///
+/// The DoG response is computed as the difference of the Gaussian responses of two images.
+///
+/// Args:
+///     src: The source image with shape (H, W).
+///     dst: The destination image with shape (H, W).
+///     sigma1: The sigma of the first Gaussian kernel.
+///     sigma2: The sigma of the second Gaussian kernel.
+pub fn dog_response_single(
```

Comment:
```suggestion
pub fn dog_response(
```
what about if we adopt this as the standard ?